sklearn/utils/validation.py

```diff
@@ -218,11 +218,12 @@ def _ensure_sparse_format(spmatrix, accept_sparse, dtype, copy,
     spmatrix : scipy sparse matrix
         Input to validate and convert.
 
-    accept_sparse : string, list of string or None (default=None)
+    accept_sparse : string, boolean or list/tuple of strings (default=False)
```

Comment:
Misleading docstring, there is no default for this function. Actually this is a problem for all the parameters in this function, can you please fix this?